exporter/signalfxexporter/testdata/json/non_default_metrics_otel_convention.json

```diff
@@ -0,0 +1,227 @@
+[
+  {
+    "system.cpu.time": null,
+    "state": "interrupt"
+  },
+  {
+    "system.cpu.time": null,
+    "state": "nice"
+  },
+  {
+    "system.cpu.time": null,
+    "state": "softirq"
+  },
+  {
+    "system.cpu.time": null,
+    "state": "steal"
+  },
+  {
+    "system.cpu.time": null,
+    "state": "system"
+  },
+  {
+    "system.cpu.time": null,
+    "state": "user"
+  },
+  {
+    "system.cpu.time": null,
+    "state": "wait"
+  },
+  {
+    "system.memory.usage": null,
+    "state": "inactive"
+  },
+  {
+    "system.filesystem.usage": null,
+    "state": "reserved"
+  },
+  {
+    "system.filesystem.inodes.usage": null
+  },
+  {
+    "system.disk.merged": null
+  },
+  {
+    "system.disk.io": null
+  },
+  {
+    "system.disk.time": null
+  },
+  {
+    "system.disk.pending_operations": null
+  },
+  {
+    "system.network.packets": null
+  },
+  {
+    "system.network.dropped": null
+  },
+  {
+    "system.network.tcp_connections": null
+  },
+  {
+    "k8s.cronjob.active_jobs": null
+  },
+  {
+    "k8s.job.active_pods": null
+  },
+  {
+    "k8s.job.desired_successful_pods": null
+  },
+  {
+    "k8s.job.failed_pods": null
+  },
+  {
+    "k8s.job.max_parallel_pods": null
+  },
+  {
+    "k8s.job.successful_pods": null
+  },
+  {
+    "k8s.statefulset.desired_pods": null
+  },
+  {
+    "k8s.statefulset.current_pods": null
+  },
+  {
+    "k8s.statefulset.ready_pods": null
+  },
+  {
+    "k8s.statefulset.updated_pods": null
+  },
+  {
+    "k8s.hpa.max_replicas": null
+  },
+  {
+    "k8s.hpa.min_replicas": null
+  },
+  {
+    "k8s.hpa.current_replicas": null
+  },
+  {
+    "k8s.hpa.desired_replicas": null
+  },
+  {
+    "k8s.container.cpu_request": null
+  },
+  {
+    "k8s.container.ephemeral-storage_limit": null
+  },
+  {
+    "k8s.container.ephemeral-storage_request": null
+  },
+  {
+    "k8s.container.memory_request": null
+  },
+  {
+    "k8s.node.condition_memory_pressure": null
+  },
+  {
+    "k8s.node.condition_network_unavailable": null
+  },
+  {
+    "k8s.node.condition_out_of_disk": null
+  },
+  {
+    "k8s.node.condition_p_i_d_pressure": null
```

Comment:
🤔 I think this came up a while back but I've forgotten the details. Is this supposed to be `pid_pressure`?

Reply:
Ah this is an artifact of a helper library we use to convert convert camel case to snake case in the k8s cluster receiver, where it converts `PIDPressure` to `p_i_d_pressure`. I guess we could tweak it so that receiver emits `pid_pressure` instead.